stock_request/views/stock_request_order_views.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record model="ir.ui.view" id="stock_request_order_tree">
+        <field name="name">stock.request.order.tree</field>
+        <field name="model">stock.request.order</field>
+        <field name="arch" type="xml">
+            <tree string="Stock requests">
+                <field name="name"/>
+                <field name="warehouse_id"
+                       groups="stock.group_stock_multi_locations"/>
+                <field name="location_id"
+                       groups="stock.group_stock_multi_locations"/>
+                <field name="state"/>
+            </tree>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="stock_request_order_form">
+        <field name="name">stock.request.order.form</field>
+        <field name="model">stock.request.order</field>
+        <field name="arch" type="xml">
+            <form string="Stock request">
+                <header>
+                    <button name="action_confirm"
+                            string="Confirm" type="object"
+                            attrs="{'invisible': [('state', 'not in', ['draft'])]}"/>
+                    <button name="action_cancel" states="draft,open"
+                            type="object" string="Cancel"/>
+                    <button name="action_draft" states="cancel" type="object"
+                            string="Set to Draft"/>
+                    <field name="state" widget="statusbar"/>
+                </header>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
```

Comment:
I suggest to add here a link to the stock requests associated to the order. It makes it easier to view than the pop-up window of the lines within the stock request order.

Reply:
Well, the lines are useful in order to add all the elements.
Do you think that is a good option to add the button once it is confirmed and hide the table then?

Reply:
@etobella Agreed. I think the lines need to be there for sure. I was asking myself if additionally the button would be useful, because you have full management capabilities in stock requests (which is different from PO lines or invoice lines). So, it boils down to usability.

Reply:
Done!